app/javascript/gobierto_data/webapp/components/sets/SQLEditor.vue

```diff
@@ -79,18 +80,26 @@ export default {
       url: '',
       endPoint: '',
       recentQueries: [],
-      newRecentQuery: null
+      newRecentQuery: null,
+      orderRecentQueries: [],
```

Comment:
You're doing the same over and over.
Do not declare variables that you won't use in the template, inside the `data()` function.
Review all the `vue` files in this PR, and remove from data all variable you don't render.